model/span.go

```diff
@@ -110,6 +111,28 @@ func (s *Span) IsRPCServer() bool {
 	return s.HasSpanKind(SpanKindServer)
 }
 
+// GetHashTag returns hash keyValue from Tags and wether the hash was found
+func (s *Span) GetHashTag() (hashCode int64, found bool) {
+	if tag, ok := KeyValues(s.Tags).FindByKey(SpanHashKey); ok {
+		return tag.GetVInt64(), true
+	}
+	return -1, false
+}
+
+func (s *Span) SetHashTag() (hashCode int64, err error) {
+	hCode, err := HashCode(s)
+	if err != nil {
+		return -1, err
+	}
+	spanHash := uint64ToInt64Bits(hCode)
+	s.Tags = append(s.Tags, KeyValue{
```

Comment:
we have helper methods for creating typed instances of KeyValue, please use that instead of explicit creation.